schema/artwork/index.js

```diff
@@ -20,8 +20,8 @@ import Dimensions from "schema/dimensions"
 import EditionSet from "schema/edition_set"
 import ArtworkLayer from "./layer"
 import ArtworkLayers, { artworkLayers } from "./layers"
-import gravity from "lib/loaders/gravity"
-import savedArtworkLoader from "lib/loaders/saved_artwork"
+import gravity from "lib/loaders/legacy/gravity"
+import savedArtworkLoader from "lib/loaders/per_type/saved_artwork"
```

Comment:
Not sure what these loaders are about, importing them seems like these are being used incorrectly. These need to move into the exported per-request loaders.